helm/revproxy/templates/ingress_aws.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
     alb.ingress.kubernetes.io/group.name: {{ .Values.global.environment }}
     alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS":443}]'
     alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
+    helm.sh/resource-policy: keep
```

Comment:
What does this do?

Reply:
This is meant to keep the ingress active between deployments so we don't have to update the CNAME DNS record.